Sample/app/src/main/java/com/microsoft/projectoxford/face/samples/ui/DetectionActivity.java

```diff
@@ -68,7 +76,9 @@
 import java.text.DecimalFormat;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.LinkedHashSet;
```

Comment:
I do not think we need to use this kind of data structure.
Arrays [1] should be sufficient.

[1] https://developer.android.com/reference/java/util/Arrays.html